src/BiometryService/BiometryService.Tests.cs

```diff
@@ -0,0 +1,54 @@
+﻿#if NETSTANDARD2_0
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Logging;
+
+namespace BiometryService;
+
+/// <summary>
+/// Implementation of the <see cref="IBiometryService" /> for Tests.
+/// </summary>
+public sealed partial class BiometryService : IBiometryService
+{
+	private readonly ILogger _logger;
+
+	/// <summary>
+	/// Initializes a new instance of the <see cref="BiometryService" /> class.
+	/// </summary>
+	/// <param name="loggerFactory">The logger factory.</param>
+	public BiometryService(ILoggerFactory loggerFactory = null)
+	{
+		_logger = loggerFactory?.CreateLogger<IBiometryService>() ?? NullLogger<IBiometryService>.Instance;
+	}
+
+	public Task<string> Decrypt(CancellationToken ct, string keyName)
```

Comment:
This implementation doesn't do much. I was hoping for something that mimicks the behavior of a real implementation. You could instead store the key-value-pairs in an in-memory dictionary so that encrypting and decrypting would actually behave like a normal implementation would. It would be important however to properly document it so say that no encryption is actually done with the test implementation.